lib/state-machines/upgrade-to-v303.json

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
Do we need both the asl and json file?

Reply:
We're maintaining this convention, so that if users have ASL plugins on their IDE (VS code for ex), they could visualise the state machine there. The .json file is the one we use for importing the definition